tests/index.html

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>Dummy Tests</title>
+    <title>Ember-CLI-Blanket Demo</title>
```

Comment:
We should probably revert this, was my bad.  I got confused as to which index.html I was looking at.